source/openapi-admin-v3.yaml

```
@@ -7203,6 +7211,18 @@ components:
             names on **any collection**.
           items:
             type: string
+        indexed_queryable_fields_names:
+          type: array
+          description: |-
+            A list of field names to use as the indexed queryable field. While this property is an array, Sync 
```

Maybe a link would be useful here?
```suggestion
            A list of field names to use as the [indexed queryable field](https://www.mongodb.com/docs/atlas/app-services/sync/configure/sync-settings/#queryable-fields). Although this property is an array, Sync 
```